tests/ACL/ACLCacheWrapperTest.php

```diff
@@ -13,17 +13,16 @@
 use OCA\GroupFolders\ACL\ACLManager;
 use OCP\Constants;
 use OCP\Files\Cache\ICache;
+use PHPUnit\Framework\MockObject\MockObject;
 use Test\TestCase;
 
 /**
  * @group DB
  */
 class ACLCacheWrapperTest extends TestCase {
-	/** @var ACLManager|\PHPUnit_Framework_MockObject_MockObject */
-	private $aclManager;
-	/** @var ICache|\PHPUnit_Framework_MockObject_MockObject */
-	private $source;
-	private ?ACLCacheWrapper $cache = null;
+	private ACLManager|MockObject $aclManager;
```

Comment:
```suggestion
	private ACLManager&MockObject $aclManager;
```
Use intersection on all mocks please.
Support added in 8.1 and main branch targets Nextcloud 31 So we’re fine.